akka-parsing/src/main/scala/akka/macros/LogHelper.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.macros
```

Comment:
Hm this package breaks OSGi though. It has to be in the package that this module is exporting.
(we'd hear back that we broke people's code...)